oaknational/Oak-Web-Application
# Curric/add sidebar filters

## Description
- Added sidebar to curriculum visualiser
- Added radio buttons for threads
- Added radio buttons for year groups
- Updated filtering logic

## How to test

1. Go to https://deploy-preview-1899--oak-web-application.netlify.thenational.academy/teachers/curriculum/geography-primary/units
2. Scroll down to `Units` tab
3. For threads, you should see a title, a selected default radio button "None highlighted" and radio buttons for threads
4. For year groups, you should see a title, a selected default radio button "All" and radio buttons for years
5. Click on a thread
6. You should see the radio button container highlighted
7. You should see the units with the selected thread highlighted
8. Click on a year
9. You should only that year on the unit list on the right

NOTE: full mobile support is not yet in place

## Screenshots

How it should now look:
{screenshots}
<img width="1176" alt="Screenshot 2023-09-14 at 10 37 41" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/611ecbb3-1817-486a-90d0-ee2b34ba387e">
<img width="1176" alt="Screenshot 2023-09-14 at 10 37 51" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/2a09bc87-6501-4cab-bad1-e5864bd082a5">
<img width="1176" alt="Screenshot 2023-09-14 at 10 38 07" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/062ef940-b8da-48a3-8fbb-2cb8a1b920d0">

## Checklist
